packaging/greenboot/green.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -ex
+
+HEALTH=healthy
+
+if [ ! -f /run/ostree-booted ]; then
```

Comment:
I agree and I think we should do this now. The only difference is that we do not have `deployment_id` - can we easily find it for RPM deplyments? If not, let's just leave it empty.

Reply:
I left that comment just for information. Let's leave support for RPM-based systems to another PR. We have at least 1 ticket related to that anyway.

Reply:
OK, agreed - let @pmtk decide.

Reply:
What's the usecase for using this script in RPM based systems?

Initial intention was for ostree-based systems only because upgrades, machine boots, and greenboot are tied.
For RPM-based you could upgrade any time without rebooting the machine and greenboot only runs (automatically) just after boot.

> I see greenboot status checks on my dev vm, and we will want to do storage migration even if we don't do backups
Then I'd include that logic in `prerun` package - auto backup/restore on ostree, migration if there's version mismatch between data and binary

Reply:
Let's merge this PR and discuss RPM-based use cases separately.